### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by Anisospina**
## Summary

- TabNavigator

## Screenshots

navigation

## Trello Card

https://trello.com/c/UO0PLWce/2-navegaci%C3%B3n

### CODE REVIEW

**Review by guidoprinc — COMMENTED** [review]
Finish with the first pull request before sending the review of this one

**Review by guidoprinc — CHANGES_REQUESTED** [review]
Correct and merge #1 before sending this one! You have many repeated changes that are solved when you merge the first one

File: toDoList/src/components/Navigation/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { createAppContainer, createStackNavigator, StackActions, NavigationActions } from 'react-navigation';
+
+class HomeScreen extends React.Component {
+  render() {
+    return (
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+        <Text>Home Screen</Text>
+        <Button
+          title="Go to Details"
+          onPress={() => {
```
Comment (guidoprinc) [inline toDoList/src/components/Navigation/index.js]: extract this function outside the render

File: toDoList/src/components/Navigation/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { View, Text, Button } from 'react-native';
+import { createAppContainer, createStackNavigator, StackActions, NavigationActions } from 'react-navigation';
+
+class HomeScreen extends React.Component {
+  render() {
+    return (
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
```
Comment (guidoprinc) [inline toDoList/src/components/Navigation/index.js]: extract this into a `styles.js` file

File: toDoList/src/components/Navigation/index.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+class DetailsScreen extends React.Component {
+  render() {
+    return (
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+        <Text>Details Screen</Text>
+      </View>
+    );
+  }
+}
```
Comment (guidoprinc) [inline toDoList/src/components/Navigation/index.js]: extract this components in other files

File: toDoList/src/components/Navigation/index.js (hunk 1 above)
Comment (Anisospina) [reply toDoList/src/components/Navigation/index.js]: it is example files...

**Review by guidoprinc — APPROVED** [review]

**Pull request merged**